configs/models/ViTForClassification.yaml

```diff
@@ -0,0 +1,74 @@
+# pytorch_lightning==1.7.0
+seed_everything: 999
+trainer:
+  logger: true
+  enable_checkpointing: true
+  callbacks: null
+  default_root_dir: null
+  gradient_clip_val: null
+  gradient_clip_algorithm: null
+  num_nodes: 1
+  num_processes: null
+  devices: null
+  gpus: -1
+  auto_select_gpus: false
+  tpu_cores: null
+  ipus: null
+  enable_progress_bar: true
+  overfit_batches: 0.0
+  track_grad_norm: -1
+  check_val_every_n_epoch: 1
+  fast_dev_run: false
+  accumulate_grad_batches: null
+  max_epochs: null
+  min_epochs: null
+  max_steps: -1
+  min_steps: null
+  max_time: null
+  limit_train_batches: null
+  limit_val_batches: null
+  limit_test_batches: null
+  limit_predict_batches: null
+  val_check_interval: null
+  log_every_n_steps: 50
+  accelerator: cuda
+  strategy: null
+  sync_batchnorm: false
+  precision: 32
+  enable_model_summary: true
+  weights_save_path: null
+  num_sanity_val_steps: 0
+  resume_from_checkpoint: null
+  profiler: null
+  benchmark: null
+  deterministic: null
```

Comment:
Would setting deterministic: true make sense for reproducibility purposes?

Reply:
I guess deterministic:true is incompatible with benchmark:true
https://pytorch.org/docs/stable/notes/randomness.html

Not sure which way to go. If `benchmark:true` provides only modest performance improvements then let's use `deterministic: true`.

Reply:
This is true. I would set `benchmark: true` as default, seeing how for most development runs determinism is unnecessary. It's only when reporting results that we want to set `deterministic: true`.